apis/v1/controllers/addresses.controller.js

```diff
@@ -49,6 +49,7 @@ exports.createAddress = async (req, res, next) => {
          if(!mongoose.Types.ObjectId.isValid(user_id)) {
```

Comment:
Your use of user_id here was not defined. I recommend using user_id for the param name. (Check in the web.api file) This is because the middleware searches for that user_id for preliminary authorization. As in this case, you need to get the user_id from the req.params object before trying to access it.